app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -337,6 +337,7 @@ def parse_container_definition(container_def, pod_id)
         :name              => container_def.name,
         :image             => container_def.image,
         :image_pull_policy => container_def.imagePullPolicy,
+        :command           => container_def.command ? container_def.command.to_json : nil,
```

Comment:
@moolitayer then this would turn into:

        :command           => container_def.command ? Shellwords.join(container_def.command) : nil,